taxonomies/categories.txt

```diff
@@ -46033,6 +46040,14 @@ en:Fruit skyrs, Fruit skyr
 fr:Skyrs aux fruits, Skyr aux fruits
 nl:Skyrs met vruchten
 
+<en:Fermented dairy desserts
+fi:Viili
```

Comment:
```suggestion
en:Viili
da:Viili
de:Viili
es:Viili
fi:Viili
```

Reply:
if it's almost never translated, then we can just add xx:Viili

Reply:
I didn't put any other translation because I didn't want to favour the finnish term over the swedish one or the opposite. But I can put both in xx:

Reply:
we do need the en: entry first, even if it's the same as the xx: entry, this is just so that all ingredients entries have a en: canonical name (not exactly the case today, but we should try do to that)